justfile

```diff
@@ -224,6 +224,18 @@ get-credentials-ci:
         --name "contrast-ci" \
         --admin
 
+get-credentials-tdxbm:
+    #!/usr/bin/env bash
+    set -euo pipefail
+    tmpConfig=$(mktemp)
+    gcloud secrets versions access \
+        projects/796962942582/secrets/m50-ganondorf-kubeconf/versions/2 \
+        --out-file="$tmpConfig"
+    mergedConfig=$(mktemp)
+    export KUBECONFIG=$tmpConfig:~/.kube/config
+    kubectl config view --flatten > $mergedConfig
+    mv $mergedConfig ~/.kube/config
```

Comment:
This should respect an existing `KUBECONFIG` var.

Reply:
I can use 
```sh
export KUBECONFIG=$tmpConfig:${KUBECONFIG:-~/.kube/config}
```
for respecting it as input, but how can I respect it when writing the config? Or is it okay to always update .kube/config, even if KUBECONFIG != .kube/config?

Reply:
Made a suggestion.